docs/source/tutorial/09-write-your-first-mutation.mdx

```diff
@@ -2,43 +2,58 @@
 title: "9. Write your first mutation"
 ---
 
-In this section, you will write your first mutation to log in to the backend.
+In this section, you will write your first mutation to log in to the backend. 
+
 A mutation is used to change data on your server. Here the login mutation will create a session based on your email address.
 
 > **Note**: The way you log in to this particular server might differ from the way you log in with your own server. Login is often handled by _middleware_, or a layer totally separate from GraphQL, like [Oauth](https://en.wikipedia.org/wiki/OAuth).
 > Also note that a typical authentication flow should require a password but for this tutorial, anyone is allowed to book flights with a valid email address!
 
 
-## Prototype your mutation in GraphQL Playground
+## Prototype your mutation in Sandbox Explorer
+
+Open [your Sandbox Explorer](https://studio.apollographql.com/sandbox/explorer?endpoint=https%3A%2F%2Fapollo-fullstack-tutorial.herokuapp.com) and click on the plus symbol to add a new tab. Next, click on the Schema icon to get back to looking at your schema, and select "Mutation" to look at your list of mutations: 
+
+<img alt="The list of available mutations" class="screenshot" src="images/sandbox_schema_mutations.png"/>
 
-Open [GraphQL Playground](https://apollo-fullstack-tutorial.herokuapp.com/) and select the `login` mutation in the docs tab on the right:
+Scroll down to take a look at the `login` mutation: 
 
 <img alt="The definition of login in the schema" class="screenshot" src="images/schema_login_definition.png"/>
 
-This mutation takes a single argument, the `email` address of the person being logged in. Unlike many GraphQL operations that return objects which need to have fields selected, the `login` mutation returns only a single string.
+Click the play button to the right to open that mutation in the Explorer tab. When it opens, click the plus sign to add the operation: 
 
-Type the following mutation in GraphiQL:
+<img alt="The login mutation after initially being added" class="screenshot" src="images/explorer_added_login_mutation.png"/>
 
-```graphql
-mutation Login($email: String) {
-  login(email: $email)
-}
-```
+Sandbox Explorer tries to be helpful by adding the word `Mutation` after the word `Login`, but the Android SDK also does that for you - so you'd wind up with a `LoginMutationMutation`. Rename the operation `Login` instead: 
```

Comment:
Actually the codegen [detects this](https://github.com/apollographql/apollo-android/blob/ffcb94feeb7772dec5ddafd8d10f471ecd125ac2/apollo-compiler/src/main/kotlin/com/apollographql/apollo3/compiler/codegen/CodegenLayout.kt#L72) and doesn't let it happen :)

Reply:
Nice. I'll take out the bit referencing that name but it's still probably a better idea not to have that in the operation name.